lib/util/createFileName.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020 LG Electronics Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Return a file name based on Date.
+function createDateFileName(separator, extName) {
+    return __getDate();
+    
+    function __getDate(){
+        if (separator === null || separator === undefined) {
+            separator = "";
+        }
+    
+        if(extName === null || extName === undefined) {
+            extName = "";
+        }
+
+        const curDate = new Date();
+        const dateFormat = __pad(curDate.getFullYear(), 2)
+            +  __pad((curDate.getMonth()+1), 2)
+            + __pad(curDate.getDate(), 2)
+            + separator
+            + __pad(curDate.getHours(), 2)
+            + __pad(curDate.getMinutes(), 2)
+            + __pad(curDate.getSeconds(), 2);
+        return dateFormat + "." + extName;
```

Comment:
if(extName) {
 return dateFormat + "." + extName;
} else {
 return dateFormat;
}

It seems that line 16~18 can be removed.

Reply:
I've updated it.